src/renderers/shared/server/ReactPartialRenderer.js

```diff
@@ -484,6 +484,19 @@ class ReactDOMServerRenderer {
       if (child === null || child === false) {
         return '';
       } else {
+        if (Array.isArray(child)) {
+          var frame = {
+            children: child,
+            childIndex: 0,
+            context: context,
+            footer: '',
+          };
+          if (__DEV__) {
+            frame.debugElementStack = [];
```

Comment:
Why do we need to do this? (Note: I'm not very familiar with this code so I'm learning it too.)